lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -27,9 +27,21 @@ class AwsCompileCloudFrontEvents {
       'before:remove:remove': this.logRemoveReminder.bind(this),
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8025
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'cloudFront', {
       type: 'object',
+      properties: {
+        eventType: {
+          type: 'string',
+          enum: ['viewer-request', 'origin-request', 'origin-response', 'viewer-response'],
+        },
+        pathPattern: { type: 'string', pattern: '^([A-Za-z0-9-_.*$/~"\'@:+]|&amp;)+$' },
+        origin: {
+          oneOf: [{ type: 'string', format: 'uri' }, { type: 'object' }],
+        },
+        isDefaultOrigin: { type: 'boolean' },
+        includeBody: { type: 'boolean' },
+        behavior: { type: 'object' },
```

Comment:
I think we miss `eventType`

Reply:
it's the first property in the object :-)
https://github.com/serverless/serverless/pull/8250/files#diff-13a25836cc7975b47c0e8dab6460f4a9R33

Reply:
🤦